Evaluate SDLINCLUDE and SDLLIB just once, not for every compile.

Remove use of undefined XPMHOME variable.
--- src/sim/makefile.orig	2018-06-29 04:19:31.000000000 -0500
+++ src/sim/makefile	2022-05-16 21:58:53.000000000 -0500
@@ -6,6 +6,6 @@
 TKLIBRARY = /usr/local/lib/tk
 
-SDLINCLUDE = $$(/bin/sh ./sdl_helper cflags)
-SDLLIBS = $$(/bin/sh ./sdl_helper libs)
+SDLINCLUDE := $(shell /bin/sh sdl_helper cflags)
+SDLLIBS := $(shell /bin/sh sdl_helper libs)
 
 CC = gcc
@@ -32,5 +32,4 @@
 INCLUDES = \
 	-Iheaders \
-	-I$(XPMHOME) \
 	-I$(XINCLUDE) \
 	-I$(TCLHOME) \
